tests/README.md

```diff
@@ -40,8 +40,17 @@
   The test does the following:
   - Deploys autotune and its dependencies using the deploy script from the autotune repo
   - Modify the layer config and apply
-  - Validate if the modified config is reflected in the JSON output from listAutotuneTunables Autotune API  
+  - Validate if the modified config is reflected in the JSON output from listAutotuneTunables Autotune API
+  
+- **RM-HPO API tests**
 
+  Here we validate the RM-HPO REST API
+  The test does the following:
+  - Start HPO servies using mock script
+  - Validate HPO result for following scenarios:
+  	1. Post invalid experiments and validate the results
+  	2. Post the same experiment again with operation set to EXP_TRIAL_GENERATE_NEW and validate the result
+  	3. Post the same experiment again with the operation set to “EXP_TRIAL_GENERATE_SUBSEQUENT” and check if same experiment continues
```

Comment:
Either remove double quotes here or quote EXP_TRIAL_GENERATE_NEW in the prev line to keep it consistent

Reply:
Include this in the third scenario - We also do a post result and check if subsequent trial number is generated